installers/cubeit-installer

```diff
@@ -848,6 +844,20 @@ if [ -n "${HDINSTALL_CONTAINERS}" ]; then
 	    done
 	fi
 
+	# Configure network offset
+	net_offset=`get_prop_value_by_container $cname "net"`
+	if [ -n "$net_offset" ] && [ $net_offset -ne 1 ]; then
+	    case "$cname" in
+	    dom0|${network_vrf})
+		# Special handling (search this file for "static")
+		;;
+	    *)
+		${SBINDIR}/cube-cfg -o ${TMPMNT}/opt/container/${cname} set cube.network.ip:192.168.42.${net_offset}/24
```

Comment:
Although there are plenty of ways to break yourself with the networking configuration. If we are fixing this up again, we should add a check to ensure that this isn't colliding with our known/reserved IPs. i.e. 1->4. No one should use the numeric net offset to set dom0, netprime, etc, just the symbolic names. So we should make sure that a hard coded offset doesn't collide.

Reply:
I was actually waiting for a chance to chat about this. I am thinking we should allow 10-100 for static (offsets). What do you think?

Reply:
That should work, just a couple of other nagging issues:

  - we really shouldn't have more than 10 system containers that need static IPs, so that isn't a problem.

  - The dhclient replacement for the dynamic container IPs needs to be tweaked to avoid the range of reserved IPs. That's an existing issue, as it could clobber now (I think), but if we want to carve out 1-10, this might be a good time to fix it.

Reply:
Correct, we basically need to carve out 3 zones and then ensure everything is aware of each zone. So the static offset tests for 10-99, the dhcp server only hands out 100-255 and so on. This was all on my todo list but I will at least make the 10-99 check here before sending another series enforcing the zones across everything else.